lib/features/booking/add_inspection_booking_screen.dart

```diff
@@ -165,9 +183,14 @@ class AddInspectionBookingBody extends StatelessWidget {
                                 isRequired: true),
                             const SizedBox(height: HsDimens.spacing4),
                             HsDropDownButton(
-                                value: '15 mins',
+                                value: null,
+                                placeholder: '15 mins',
```

Comment:
use localisation

Reply:
It was done in UI story only. This will be fixed in story 395- selectingDuration